sessionctx/variable/sysvar.go

```diff
@@ -623,6 +623,8 @@ var defaultSysVars = []*SysVar{
 	{ScopeSession, TiDBOptAggPushDown, boolToIntStr(DefOptAggPushDown)},
 	{ScopeGlobal | ScopeSession, TiDBBuildStatsConcurrency, strconv.Itoa(DefBuildStatsConcurrency)},
 	{ScopeGlobal, TiDBAutoAnalyzeRatio, strconv.FormatFloat(DefAutoAnalyzeRatio, 'f', -1, 64)},
+	{ScopeGlobal, TiDBAutoAnalyzeStartTime, DefAutoAnalyzeStartTime},
```

Comment:
To safely use this feature, all the tidb-servers in a same cluster is required to be set in the same time zone?

Reply:
never mind, the only thing that the DBAs need to do is to guarantee that the all the tidb-servers in the cluster can be used to analyze tables in the given time period based on the given time zone.